ssl/tls13_enc.cc

```diff
@@ -369,6 +369,21 @@ bool tls13_finished_mac(SSL_HANDSHAKE *hs, uint8_t *out, size_t *out_len,
   return true;
 }
 
+bool tls13_finished_mac_pha(SSL *ssl, uint8_t *out, size_t *out_len) {
```

Comment:
Another function where we could abstract the code from `tls13_finished_mac` into a static function to share with this.